beesdoo_inventory/__manifest__.py

```diff
@@ -11,11 +11,10 @@
     "author": "Beescoop - Cellule IT, Coop IT Easy SC",
     "website": "https://github.com/beescoop/Obeesdoo",
     "category": "Inventory",
-    "version": "12.0.1.0.1",
+    "version": "12.0.2.0.1",
```

Comment:
No need to bump. ocabot does it for us.

Reply:
I need to do it to control my migration scripts. I'll _nobump merge_ it (or _patch merge_ it ?) .

Reply:
ahh makes sense. `merge nobump` is ok